components/eam/src/physics/cam/micro_p3.F90

```diff
@@ -2855,6 +2862,125 @@ subroutine prevent_ice_overdepletion(pres,t_atm,qv,latent_heat_sublim,inv_dt,
 
 end subroutine prevent_ice_overdepletion
 
+subroutine water_vapor_conservation(qv,qidep,qinuc,qi2qv_sublim_tend,qr2qv_evap_tend,dt)
```

Comment:
Can we get a one-line comment about exactly what this subroutine does and when to call it?

Reply:
Done.